lnst/Recipes/ENRT/UseVfsMixin.py

```diff
@@ -33,17 +39,22 @@ def test_wide_configuration(self):
             for dev_name in dev_names:
                 dev = getattr(host, dev_name)
                 sriov_devices = SRIOVDevices(dev, 1)
+
                 vf_dev = sriov_devices.vfs[0]
                 host.map_device(dev_name, {"ifname": vf_dev.name})
 
-                host_config = config.vf_config.setdefault(host, [])
+                host_config = self.vf_config.setdefault(host, [])
                 host_config.append(sriov_devices)
 
+        if self.params.get("vf_trust"):
```

Comment:
the `vf_trust` parameter will always have either an `"on"` or `"off"` value so this condition will always be True... maybe you wanted to check `if self.params.use_vfs` but that's also not necessary since you check that higher in the method.

Reply:
ah ok, it doesn't have a default, and parameters are optional by default... ok